fe/fe-core/src/main/java/com/starrocks/meta/lock/Locker.java

```diff
@@ -14,37 +14,71 @@
 
 package com.starrocks.meta.lock;
 
+import com.google.common.base.Objects;
 import com.starrocks.catalog.Database;
 import com.starrocks.catalog.Table;
 import com.starrocks.common.Config;
+import com.starrocks.common.ErrorCode;
+import com.starrocks.common.ErrorReportException;
+import com.starrocks.server.GlobalStateMgr;
 
 import java.util.concurrent.TimeUnit;
 
 public class Locker {
```

Comment:
If many locks should be hold in one thread. Should multiple lockers need to be executed separately? or only one locker is required to perform all lock operations?

Reply:
Can you give a definition for Locker

Reply:
To be precise, the current Locker should be ThreadLocker. Each Thread has only one Locker. Multiple New Lockers use thread id as the unique identifier.